src/components/card/Description.vue

```diff
@@ -175,8 +176,33 @@ export default {
 			}
 			this.descriptionEditing = true
 			this.description = this.card.description
+
+			// Has to start after the Editor is fully loaded. This shouldn't take longer than 1/4 second
+			setTimeout(() => {
```

Comment:
To avoid the setTimeout you could also register this on a callback of the initialized event that the VueEasyMDE component emits

Reply:
Thanks for the feedback, i just added a method that's called via `@initialized`. The Code inside the function is exactly the same as in the previous version :)